geode-core/src/main/java/org/apache/geode/redis/internal/Pair.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package org.apache.geode.redis.internal;
+
+public class Pair<T, U> {
+  T fst;
+  U snd;
+
+  Pair(Object fst, Object snd) {
```

Comment:
I think you should just use `Pair(T fst, U snd)` here and in `of`.

Reply:
Ah ok, sounds good.

Reply:
You can! Try:
```
  public static <T,U> Pair<T,U> of(T fst, U snd) {
    return new Pair<>(fst, snd);
  }
```